LOAN/LoanContracts/LoanApplications.rdf

```diff
@@ -157,14 +368,200 @@
 		<rdfs:label xml:lang="en">loan at application stage</rdfs:label>
 	</owl:Class>
 	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-app;ManualUnderwriting">
+		<rdf:type rdf:resource="&fibo-loan-ln-app;UnderwritingAutomation"/>
+		<rdfs:label>manual underwriting</rdfs:label>
+		<skos:definition>underwriting was performed manually</skos:definition>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PreApprovalContract">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-agr-ctr;WrittenContract"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;hasInterestRate"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;InterestRate"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;hasApprovedAmount"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>pre-approval contract</rdfs:label>
+		<skos:definition>written commitment to lend when specified conditions are met, such as finding suitable property, and unchanged creditworthiness</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>A commitment from a lender to a borrower to extend a home purchase loan up to a certain amount, and subject to certain non-credit related conditions.  This commitment is entered into after a comprehensive analysis of the credit worthiness of the borrower is carried out.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>This may also include limits on the region where to purchase.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PreApprovalRequest">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Request"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;hasBorrower"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;Borrower"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-app;hasRequestedAmount"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>pre-approval request</rdfs:label>
+		<skos:definition>request from a potential borrower that a lender commit to pre-approving the borrower for a loan of up to a specified amount of money</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>This may also include limits on the region where to purchase.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-ln-app;PublicRecord">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-doc;Record"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;refersTo"/>
```

Comment:
I think this is related to records search results when doing credit risk analysis, and may reference a credit card account or something else, but otherwise I would defer to an SME.  I don't think there is always an account involved in any case, and made this optional when I moved it.